<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qhome</class>
 <widget class="QWidget" name="Qhome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/1.jpg);</string>
  </property>
  <widget class="QPushButton" name="writeButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;System&quot;;
background-color: rgba(12, 57, 134, 200);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButtton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;System&quot;;
background-color: rgba(12, 57, 134, 200);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="receiving">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 93, 250, 0);
background-image: url(:/new/prefix1/receiving4.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="receiveButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;System&quot;;
background-color: rgba(12, 57, 134, 200);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Receive</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="writing">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 93, 250, 0);
background-image: url(:/new/prefix1/writing4.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
